pages/common/g++.md

```diff
@@ -0,0 +1,19 @@
+# G Plus Plus
+
+> G++ is a compiler for C++. Member of GNU GCC.
+
+- Compile a source code file into an executable binary:
+
+`g++ {{input_source.cpp}} -o {{output_executable}}`
+
+- Activate output of (almost) all errors and warnings:
+
+`g++ {{input_source.cpp}} -Wall -o {{output_executable}}`
+
+- Choose a language standard to compile for(C++98/C++11/C++14/C++17):
+
+`g++ {{input_source.cpp}} -std={{language_standard}} -o {{output_executable}}`
```

Comment:
Change language_standard to `C++98|C++11|C++14|C++17`

Reply:
There are many other C++ standards, it's hard to create an exhaustive list here.

Reply:
Ah ok, ignore it then.